recipes/catx/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "catx" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/catx-{{ version }}.tar.gz
+  sha256: 31c8b46ae6c8ffe39fe95ef40106e594780a95190ad83c13d28f22b295a4ae44
+
+build:
+  skip: True  # [win or py<37]
```

```suggestion
  entry_points:
    - project_name = project_name.__main__:main
  noarch: python
```
This allows the package to be noarch. The missing tensorflow dependency can be ignored as the solver will inform the user that it is not available during installation.